pkg/client/connector/install.go

```diff
@@ -273,6 +273,31 @@ func (ki *installer) findMatchingServices(portName string, dep *kates.Deployment
 	return matching
 }
 
+func (ki *installer) findMatchingServicesByLabels(portName string, labels map[string]string) []*kates.Service {
```

Comment:
This goes away in future commits, decided to just meld it into findMatchingServices, so you can ignore it for now